<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CyTOF panel</string>
  </property>
  <widget class="QPushButton" name="Home_btn">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>46</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>3</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CyTOF panel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="panel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>861</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragOnly</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::CopyAction</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="sheet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>53</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Com_btn">
   <property name="geometry">
    <rect>
     <x>532</x>
     <y>46</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Company panel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ex_btn">
   <property name="geometry">
    <rect>
     <x>672</x>
     <y>46</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Experiment panel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
